### PR TITLE
Update version info about default enabled Placement Rules

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
I have updated version info about default enabled Placement Rules in "configure-placement-rules.md" because this feature is enabled by default since TiDB v 5.0.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/issues/7239

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: configure-placement-rules.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Placement Rules 示意图如下所示：
 
 ### 开启 Placement Rules 特性
 
-默认情况下，Placement Rules 特性是关闭的。要开启这个特性，可以集群初始化以前设置 PD 配置文件：
+Placement Rules 特性在 TiDB v5.0 及以上的版本中是默认开启的。如需手动开启这个特性，可以[集群初始化以前设置 PD 配置文件](/###关闭-placement-rules-特性)：
```
[the remark this answers is not in the document]
Comment (en-jin19) [reply configure-placement-rules.md:74]:
```suggestion
Placement Rules 特性在 TiDB v5.0 及以上的版本中默认开启。如需关闭 Placement Rules 特性，请参考[关闭 Placement Rules] (/###关闭-placement-rules-特性)。如需在关闭后重新开启该特性，可以集群初始化以前设置 PD 配置文件：
```

[the remark this answers is not in the document]
Comment (nolouch) [reply configure-placement-rules.md:74]: 如需在关闭后重新开启该特性，可以动态开启的。

[the remark this answers is not in the document]
Comment (disksing) [reply configure-placement-rules.md:74]: 还有开头的“Placement Rules 是 PD 在 4.0 版本引入的试验特性”也一起改一下吗？都默认打开了，肯定不能算实验特性了

**Review by nolouch — APPROVED** [review]

File: configure-placement-rules.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ aliases: ['/docs-cn/dev/configure-placement-rules/','/docs-cn/dev/how-to/configu
 
 Placement Rules 是 PD 在 4.0 版本引入的试验特性，它是一套副本规则系统，用于指导 PD 针对不同类型的数据生成对应的调度。通过组合不同的调度规则，用户可以精细地控制任何一段连续数据的副本数量、存放位置、主机类型、是否参与 Raft 投票、是否可以担任 Raft leader 等属性。
 
+Placement Rules 特性在 TiDB v5.0 及以上的版本中默认开启。如需关闭 Placement Rules 特性，请参考[关闭 Placement Rules] (/###关闭-placement-rules-特性)。
```
Comment (en-jin19) [inline configure-placement-rules.md:15]:
```suggestion
Placement Rules 特性在 TiDB v5.0 及以上的版本中默认开启。如需关闭 Placement Rules 特性，请参考[关闭 Placement Rules](/###关闭-placement-rules-特性)。
```

**Review by qiancai — APPROVED** [review]

File: configure-placement-rules.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (en-jin19) [reply configure-placement-rules.md:76]: @nolouch 动态开启是指通过集群初始化以前设置 PD 配置文件动态开启的意思吗？

[the remark this answers is not in the document]
Comment (en-jin19) [reply configure-placement-rules.md:76]: @disksing 好的，那开头的第一句话改成“Placement Rules 是 PD 在 4.0 版本引入的特性”，把”试验“两字删掉吧

File: configure-placement-rules.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ aliases: ['/docs-cn/dev/configure-placement-rules/','/docs-cn/dev/how-to/configu
 
 Placement Rules 是 PD 在 4.0 版本引入的试验特性，它是一套副本规则系统，用于指导 PD 针对不同类型的数据生成对应的调度。通过组合不同的调度规则，用户可以精细地控制任何一段连续数据的副本数量、存放位置、主机类型、是否参与 Raft 投票、是否可以担任 Raft leader 等属性。
```
Comment (en-jin19) [inline configure-placement-rules.md:13]:
```suggestion
Placement Rules 是 PD 在 4.0 版本引入的特性，它是一套副本规则系统，用于指导 PD 针对不同类型的数据生成对应的调度。通过组合不同的调度规则，用户可以精细地控制任何一段连续数据的副本数量、存放位置、主机类型、是否参与 Raft 投票、是否可以担任 Raft leader 等属性。
```

File: configure-placement-rules.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (qiancai) [reply configure-placement-rules.md:76]: https://docs.pingcap.com/zh/tidb/stable/experimental-features#%E8%B0%83%E5%BA%A6%E5%8A%9F%E8%83%BD 也提到了 Placement Rules 是 v4.0 实验特性。

[the remark this answers is not in the document]
Comment (nolouch) [reply configure-placement-rules.md:76]: 这个可以再 placement rule in sql GA 之后一起去掉吧

[the remark this answers is not in the document]
Comment (en-jin19) [reply configure-placement-rules.md:76]: 好哒，目前在本文中先保留这部分，等下周 Placement Rules in SQL GA 那个中英文 PR 合并后，再把所有文档中的“Placement Rules 是 PD 在 4.0 版本引入的功能”句子统一去掉吧，大家看呢？

File: configure-placement-rules.md (hunk 3 above)
Comment (qiancai) [reply configure-placement-rules.md:13]: 这个改动可能没有必要，因为它的确是实验特性

File: configure-placement-rules.md [hunk 4]
```diff
@@ -80,7 +82,7 @@ Placement Rules 示意图如下所示：
 enable-placement-rules = true
 ```
 
-这样，PD 在初始化成功后会开启这个特性，并根据 `max-replicas` 及 `location-labels` 配置生成对应的规则：
+这样，PD 在初始化成功后会开启这个功能，并根据 `max-replicas` 及 `location-labels` 配置生成对应的规则：
```
Comment (en-jin19) [inline configure-placement-rules.md:85]:
```suggestion
这样，PD 在初始化成功后会开启这个特性，并根据 `max-replicas` 及 `location-labels` 配置生成对应的规则：
```

File: configure-placement-rules.md [hunk 5]
```diff
@@ -111,9 +113,9 @@ PD 同样将根据系统的 `max-replicas` 及 `location-labels` 生成默认的
 >
 > 开启 Placement Rules 后，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
 
-### 关闭 Placement Rules 特性
+### 关闭 Placement Rules 功能
```
Comment (en-jin19) [inline configure-placement-rules.md:116]:
```suggestion
### 关闭 Placement Rules 特性
```

File: configure-placement-rules.md [hunk 6]
```diff
@@ -111,9 +113,9 @@ PD 同样将根据系统的 `max-replicas` 及 `location-labels` 生成默认的
...
 
-使用 pd-ctl 可以关闭 Placement Rules 特性，切换为之前的调度策略。
+使用 pd-ctl 可以关闭 Placement Rules 功能，切换为之前的调度策略。
```
Comment (en-jin19) [inline configure-placement-rules.md:118]:
```suggestion
使用 pd-ctl 可以关闭 Placement Rules 特性，切换为之前的调度策略。
```

File: configure-placement-rules.md [hunk 7]
```diff
@@ -69,9 +71,9 @@ Placement Rules 示意图如下所示：
 
 本节的操作步骤以使用 [pd-ctl](/pd-control.md) 工具为例，涉及到的命令也支持通过 HTTP API 进行调用。
 
-### 开启 Placement Rules 特性
+### 开启 Placement Rules 功能
```
Comment (en-jin19) [inline configure-placement-rules.md:74]:
```suggestion
### 开启 Placement Rules 特性
```

File: configure-placement-rules.md (hunk 4 above)
Comment (en-jin19) [inline configure-placement-rules.md]:
```suggestion
这样，PD 在初始化成功后会开启这个特性，并根据 `max-replicas` 及 `location-labels` 配置生成对应的规则：
```

**Review by nolouch — APPROVED** [review]

File: configure-placement-rules.md [hunk 8]
```diff
@@ -12,6 +12,8 @@ aliases: ['/docs-cn/dev/configure-placement-rules/','/docs-cn/dev/how-to/configu
 
 Placement Rules 是 PD 在 4.0 版本引入的试验特性，它是一套副本规则系统，用于指导 PD 针对不同类型的数据生成对应的调度。通过组合不同的调度规则，用户可以精细地控制任何一段连续数据的副本数量、存放位置、主机类型、是否参与 Raft 投票、是否可以担任 Raft leader 等属性。
 
+Placement Rules 特性在 TiDB v5.0 及以上的版本中默认开启。如需关闭 Placement Rules 特性，请参考[关闭 Placement Rules](/###关闭-placement-rules-特性)。
```
Comment (TomShawn) [inline configure-placement-rules.md:15]:
```suggestion
Placement Rules 特性在 TiDB v5.0 及以上的版本中默认开启。如需关闭 Placement Rules 特性，请参考[关闭 Placement Rules](#关闭-placement-rules-特性)。
```

File: configure-placement-rules.md [hunk 9]
```diff
@@ -71,7 +73,7 @@ Placement Rules 示意图如下所示：
 
 ### 开启 Placement Rules 特性
 
-默认情况下，Placement Rules 特性是关闭的。要开启这个特性，可以集群初始化以前设置 PD 配置文件：
+Placement Rules 特性在 TiDB v5.0 及以上的版本中默认开启。如需关闭 Placement Rules 特性，请参考[关闭 Placement Rules](/###关闭-placement-rules-特性)。如需在关闭后重新开启该特性，可以集群初始化以前设置 PD 配置文件：
```
Comment (TomShawn) [inline configure-placement-rules.md:76]:
```suggestion
Placement Rules 特性在 TiDB v5.0 及以上的版本中默认开启。如需关闭 Placement Rules 特性，请参考[关闭 Placement Rules](#关闭-placement-rules-特性)。如需在关闭后重新开启该特性，可以集群初始化以前设置 PD 配置文件：
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**